src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import duke.command.Command;
+import duke.task.TaskList;
+
+/**
+ * A personal assistant chat bot to help users keep track of tasks.
+ */
+public class Duke {
+    
+    private final UI ui;
+    private final TaskList taskList;
+    private final Storage storage;
+    private final Path filePath;
+
+    /**
+     * Creates a new instance of the Duke chat bot with the default filepath.
+     */
+    public Duke() {
+        this(Paths.get("data", "duke.txt"));
+    }
+
+    /**
+     * Creates a new instance of the Duke chat bot with a custom filepath.
+     * @param filePath File path where Duke should read from and save to.
```

Comment:
Very minor style violation here, but I think there should be an empty line between the description and parameter.